app/models/user.rb

```diff
@@ -2,6 +2,11 @@ class User < ApplicationRecord
   include CloudinaryHelper
 
   attr_accessor :scholar_email
+  attr_accessor :add_mentor
+  attr_accessor :add_mentee
+  attr_accessor :mentorship_note
+  attr_accessor :ban_from_mentorship
```

Comment:
You can keep them all in oneline, like so: `attr_accesor :who, :what, :where, :when, :why`